src/main/steps/applicant1/hub-page/right-column/content.ts

```diff
@@ -47,8 +49,12 @@ export const generateContent: TranslationFn = content => {
   const statesWithoutRespondentAnswers = [State.AwaitingAos, State.AosDrafted, State.AosOverdue];
   const aosSubmitted =
     !content.isJointApplication && !statesWithoutRespondentAnswers.includes(<State>content.userCase.state);
+  const hasCertificateOfService = content.userCase.alternativeServiceOutcomes?.find(
+    alternativeServiceOutcome => alternativeServiceOutcome.value.successfulServedByBailiff === YesOrNo.YES
```

Comment:
there's quite a lot of overlap with our two prs, as I used 
  const hasCertificateOfService =
    content.userCase.alternativeServiceOutcomes?.length &&
    content.userCase.alternativeServiceOutcomes?.[0].value.alternativeServiceType !== AlternativeServiceType.BAILIFF;
 
just to note that one of us will have to sort out the conflicts for it

Reply:
also your .find() works better so I'll update my pr to use that

Reply:
thats fine, we can clean it up based on which ever branch is in first. i can clean up if you want. main issue, why is CERTIFICATE_OF_SERVICE common, is it a different document with the same name or the same document, if its the same, then we just gotta show it for any condition tbh as long as the document exist?